src/bun.js/webcore/blob.zig

```diff
@@ -2513,6 +2537,23 @@ pub const Blob = struct {
         return ZigString.Empty.toValue(globalThis);
     }
 
+    pub fn getLastModified(
+        this: *Blob,
+        _: *JSC.JSGlobalObject,
+    ) callconv(.C) JSValue {
+        if (this.store) |store| {
+            if (store.data == .file) {
+                // if lastModified time is not 0, it can be already set during read.
+                if (store.data.file.last_modified == 0) {
```

Comment:
We can merge without this but

It would be better to use a non-zero value as the default because I'm not sure if special files (such as `/proc/cpuinfo`) will set a last modified at. It would be best to avoid calling `stat()` on those when the values won't change. Setting it to `std.math.maxInt(u52)` would let us distinguish between a value which hasn't been without occupying additional space using an `?` optional

Reply:
Sure. Will update to maxInt(u52). 

BTW, cpuinfo has some timestamp attached. Anyway, there could be some other special file does not have timestamp.:
```
stat -L /proc/cpuinfo 
  File: ‘/proc/cpuinfo’
  Size: 0         	Blocks: 0          IO Block: 1024   regular empty file
Device: 5h/5d	Inode: 4026532402  Links: 1
Access: (0444/-r--r--r--)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 2023-03-27 23:34:07.318464837 +0000
Modify: 2023-03-27 23:34:07.318464837 +0000
Change: 2023-03-27 23:34:07.318464837 +0000
 Birth: -
```

Reply:
Is there any update for above question?

Reply:
looks fine